doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -299,6 +299,8 @@ nRF Desktop
   * The :ref:`nrf_desktop_dfu_mcumgr` to recognize the MCUmgr custom group ID (:kconfig:option:`CONFIG_MGMT_GROUP_ID_SUIT`) from the SUITFU subsystem (:kconfig:option:`CONFIG_MGMT_SUITFU`) as a DFU-related command group.
   * All build configurations with the DFU over MCUmgr support to require encryption for operations on the Bluetooth GATT SMP service (see the :kconfig:option:`CONFIG_MCUMGR_TRANSPORT_BT_PERM_RW_ENCRYPT` Kconfig option).
     The Bluetooth pairing procedure of the unpaired Bluetooth peers must now be performed before the DFU operation.
+  * The :ref:`nrf_desktop_dfu_mcumgr` to enable the bootloader information hook that is intended for SUIT-based DFU.
```

Comment:
Please also inform that MCUboot has bootloader information hook enabled too

Reply:
added a dedicated changelog for this part